package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-scm-base",
-  "version": "2.4.0",
+  "version": "3.0.0",
```

Comment:
I don't think so. This is adding `getOpenedPRs` to the required methods. If we don't bump major version and rebuild `scm-bitbucket` or `scm-github` (which will pull in this version of scm-base), it will fail at `Not implemented`.